client/my-sites/plans-grid/hooks/npm-ready/data-store/use-grid-plans.tsx

```diff
@@ -346,7 +335,7 @@ const useGridPlans = ( {
 
 		return {
 			planSlug,
-			isVisible: planSlugsForIntent.includes( planSlug ),
+			isVisible: true,
```

Comment:
`Non-blocking / nitpick`:

Just clarifying for my own understanding, but it seems like we'll be able remove the `isVisible` property if we do, in fact, only need to evaluate `planSlugsForIntent`. Are the references to `isVisible` in the comparison grid something we'll clean up in a later PR?

Reply:
Great question/ good thinking. I feel we should remove it if this change works out nicely.
I will trace it out and see if it's something we can do right away or wait until the hooks are in more final form. 

In any case, I will put this behind an issue for a follow-up